aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -18,6 +18,11 @@
 #ifdef USE_ROCM
 #define PYTORCH_ROCBLAS_VERSION_DECIMAL (ROCBLAS_VERSION_MAJOR * 100 + ROCBLAS_VERSION_MINOR)
 #define USE_GEMM_FLAGS_FP16_ALT_IMPL (PYTORCH_ROCBLAS_VERSION_DECIMAL >= 242)
+
+#if ROCM_VERSION >= 50200
```

Comment:
Should be `#if defined(USE_ROCM) && ROCM_VERSION >= 50200`.

Reply:
This is already surrounded by `#ifdef ROCM` (Line 18 above). Must we repeat?

Reply:
Good point, not needed.